h/static/scripts/annotator/plugin/bucket-bar.coffee

```diff
@@ -4,7 +4,7 @@ $ = Annotator.$
 
 class Annotator.Plugin.BucketBar extends Annotator.Plugin
   # prototype constants
-  BUCKET_THRESHOLD_PAD: 30
+  BUCKET_THRESHOLD_PAD: 105
```

Comment:
I just myself made this 106 on master. Could you just rebase on that and this PR doesn't need to concern itself.